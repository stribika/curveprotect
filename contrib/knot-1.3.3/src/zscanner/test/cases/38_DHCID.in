$ORIGIN	.
$TTL	1

; OK
@	DHCID	AA==		; The simplest variant
@	DHCID	Zm8=		; One char padding
@	DHCID	Zm9v		; Without padding
@	DHCID	Zm9vYg==	; Two base64 blocks
@	DHCID	Zm9v YmE=	; Two base64 blocks with blank space between them
@	TYPE49	\# 1 00		; TYPE + Hexadecimal rdata
@	TYPE49	AA==		; TYPE
@	dhcid	AA==		; Type in lower-case

; KO
@	DHCID
@	DHCID			; Empty rdata
@	DHCID	\# 0		; Hex empty rdata
@	DHCID	\#		; Missing hex length
@	DHCID	A		; Continuous block length must be multiple of 4
@	DHCID	AB		; Continuous block length must be multiple of 4
@	DHCID	ABC		; Continuous block length must be multiple of 4
@	DHCID	AA ==		; Continuous block length must be multiple of 4
@	DHCID	A===		; Bad padding
@	DHCID	=		; Bad padding
@	DHCID	==		; Bad padding
@	DHCID	===		; Bad padding
@	DHCID	====		; Bad padding
@	DHCID	\# 1 0000	; Too long rdata
@	DHCID	\# 2 00		; Bad rdata length
