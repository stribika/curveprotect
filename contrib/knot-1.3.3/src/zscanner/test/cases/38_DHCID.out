OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=666F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=666F6F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=666F6F62
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=666F6F6261
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0031
RDATA=00
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_BASE64_CHAR
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
